src/components/ManageAccountsDrawer/components/LedgerOpenApp.tsx

```diff
@@ -0,0 +1,134 @@
+import { Button } from '@chakra-ui/react'
+import {
+  arbitrumChainId,
+  arbitrumNovaChainId,
+  avalancheChainId,
+  bchChainId,
+  binanceChainId,
+  bscChainId,
+  btcChainId,
+  type ChainId,
+  cosmosChainId,
+  dogeChainId,
+  ethChainId,
+  gnosisChainId,
+  ltcChainId,
+  optimismChainId,
+  polygonChainId,
+  thorchainChainId,
+} from '@shapeshiftoss/caip'
+import type { slip44Table } from '@shapeshiftoss/hdwallet-core'
+import { isLedger } from '@shapeshiftoss/hdwallet-ledger'
+import { useCallback, useEffect } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { selectFeeAssetByChainId } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { DrawerContentWrapper } from './DrawerContent'
+
+export type LedgerOpenAppProps = {
+  chainId: ChainId
+  onClose: () => void
+  onNext: () => void
+}
+
+type Slip44Key = keyof typeof slip44Table
+const getSlip44KeyFromChainId = (chainId: ChainId): Slip44Key | undefined => {
+  switch (chainId) {
+    // UTXO chains
+    case btcChainId:
+      return 'Bitcoin'
+    case dogeChainId:
+      return 'Dogecoin'
+    case bchChainId:
+      return 'BitcoinCash'
+    case ltcChainId:
+      return 'Litecoin'
+    // EVM chains
+    case ethChainId:
+      return 'Ethereum'
+    case avalancheChainId:
+      return 'Avalanche'
+    case optimismChainId:
+      return 'Optimism'
+    case bscChainId:
+      return 'BnbSmartChain'
+    case polygonChainId:
+      return 'Polygon'
+    case gnosisChainId:
+      return 'Gnosis'
+    case arbitrumChainId:
+      return 'Arbitrum'
+    case arbitrumNovaChainId:
+      return 'ArbitrumNova'
+    // Cosmos chains
+    case thorchainChainId:
+      return 'Rune'
+    case cosmosChainId:
+      return 'Atom'
+    case binanceChainId:
+      return 'Binance'
+    default:
+      return undefined
```

Comment:
Do we want to `assertUnreachable` here to make things easier when we add new chain?
Or alternatively, have an exhaustive mapping of `<ChainId, string>` to ensure we error our at type-level when adding new chains and don't forget to add this here

Reply:
Agree, though I'm going to do this in a follow-up PR.